tools/installers/install_lightning.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+if ! python -c "import packaging.version" &> /dev/null; then
+    python3 -m pip install packaging
+fi
+torch_version=$(python3 -c "import torch; print(torch.__version__)")
+
+echo "[INFO] torch_version=${torch_version}"
+
+cat >> lightning_constraints.txt << EOF
+torch==${torch_version}
+EOF
+
+python3 -m pip install -c lightning_constraints.txt lightning
+
```

Comment:
@sw005320 @popcornell Looks like this simplified version works. We get the torch version and set it as a constraint file to `pip install lightning`. The constraint file differs from a requirements file, so `torch` will be used as a constraint but will not be installed.